common/views/components/IIIFViewerPrototype/IIIFViewerPrototype.tsx

```diff
@@ -73,87 +74,116 @@ const ZoomedImagePrototype = dynamic(
-    height: 1px;
-    margin: -1px;
-    overflow: hidden;
-    padding: 0;
-    position: absolute;
-    width: 1px;
-    white-space: nowrap;
-  }
-`;
-
 const Grid = styled.div`
   display: grid;
   height: calc(100vh - 85px); // FIXME: use variable for header height
   overflow: hidden;
   grid-template-columns: [left-edge] minmax(200px, 330px) [desktop-sidebar-end main-start desktop-topbar-start] 9fr [right-edge];
-  grid-template-rows: [top-edge] min-content [desktop-main-start desktop-topbar-end] 1fr [bottom-edge];
+  grid-template-rows: [top-edge] min-content [desktop-main-start desktop-topbar-end] 1fr [mobile-bottombar-start mobile-main-end] min-content [bottom-edge];
 `;
 
-const Sidebar = styled.div<{ isActive: boolean }>`
-  display: ${props => (props.isActive ? 'inherit' : 'none')};
-  grid-area: top-edge / left-edge / bottom-edge / desktop-sidebar-end;
+const Sidebar = styled.div<{
+  isActiveMobile: boolean;
+  isActiveDesktop: boolean;
+  isMobile: boolean;
+}>`
+  display: ${props =>
+    (props.isActiveMobile && props.isMobile) ||
+    (props.isActiveDesktop && !props.isMobile)
+      ? 'inherit'
+      : 'none'};
+  grid-area: top-edge / left-edge / bottom-edge /
+    ${props => (props.isMobile ? ' right-edge ' : 'desktop-sidebar-end')};
   background: ${props => props.theme.color('viewerBlack')};
   color: ${props => props.theme.color('white')};
   border-right: 1px solid ${props => props.theme.color('charcoal')};
   overflow: auto;
+  z-index: 5;
 `;
 
-const Topbar = styled.div<{ isSidebarActive: boolean }>`
+const Topbar = styled.div<{
+  isMobileSidebarActive: boolean; // TODO: required here?
+  isDesktopSidebarActive: boolean;
+  isMobile: boolean;
+}>`
   background: ${props => props.theme.color('charcoal')};
-  grid-area: top-edge /
-    ${props => (props.isSidebarActive ? 'desktop-topbar-start' : 'left-edge')} /
-    desktop-topbar-end / right-edge;
-  z-index: 4;
+  z-index: 5;
+
+  ${props =>
+    props.isMobile &&
+    `
+    z-index: 4; // TODO: this is to let downloads sit above sidebar on desktop but not have the topbar above the sidebar on mobile. If we move the downloads, this can be simplified
+    grid-area: top-edge / left-edge / desktop-topbar-end / right-edge;
+  `}
+
+  ${props =>
+    !props.isMobile &&
+    `
+    grid-area: top-edge / ${
+      props.isDesktopSidebarActive ? 'desktop-topbar-start' : 'left-edge'
+    } / desktop-topbar-end / right-edge;
+  `}
 `;
 
-const Main = styled.div<{ isSidebarActive: boolean }>`
+const Main = styled.div<{
+  isMobileSidebarActive: boolean; // TODO: required here?
+  isDesktopSidebarActive: boolean;
+  isMobile: boolean;
+}>`
   background: ${props => props.theme.color('viewerBlack')};
   color: ${props => props.theme.color('white')};
-  grid-area: desktop-main-start /
-    ${props => (props.isSidebarActive ? 'main-start' : 'left-edge')} /
-    bottom-edge / right-edge;
   overflow: auto;
   position: relative;
+
+  ${props =>
+    props.isMobile &&
```

Comment:
Shouldn't we be using media queries not JS here?

Reply:
I had a chat with @gestchild about this, and we settled on keeping it all in JS since it is coupled to whether we render bits of ui in certain places based on whether we're on mobile/desktop.

Reply:
Would this break once we get it rendering without JS though?

Reply:
It would essentially mean you get the desktop experience if you don't have JS.

Reply:
And we can avoid this by using just CSS? I am not sure I understand the logic to do some responsive styling in JS, and some in CSS?

Reply:
I see what you mean. I think in that case, in this instance I would favour moving this logic exclusively into the template (JS). Because we are necessarily going to render certain elements in entirely different parts of the screen based on how big your screen is (not really addressable by media queries) as a UX decision, keeping all the logic in the one place (JS) feels like it would make sense?

Reply:
One of the big problems I see with this is jank, which we already have a problem with (#6268) especially around the search form.

Could we not just have 1 simple css query of
```
.desktop-viewer {
  display: none
  desktop-breakpoint {
    display: block
  }
}
.mobile-viewer {
  display: block
  desktop-breakpoint {
    display: none
  }
}
```
?

Reply:
yeah, that would probably make more sense, at the expense of having to have extra elements in the DOM. I'll update with that. Probably scoping the `.desktop-viewer` and `.mobile-viewer` classes to the parent `IIIFViewer` top-level styled component?